web/src/components/atomic/Icon.vue

```diff
@@ -15,12 +15,12 @@
   <SvgIcon v-else-if="name === 'settings'" :path="mdiCog" size="1.3rem" />
   <SvgIcon v-else-if="name === 'settings-outline'" :path="mdiCogOutline" size="1.3rem" />
   <SvgIcon v-else-if="name === 'trash'" :path="mdiTrashCanOutline" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'status-blocked'" :path="mdiPlayOutline" size="1.3rem" />
-  <SvgIcon v-else-if="name === 'status-declined'" :path="mdiStopCircleOutline" size="1.3rem" />
+  <SvgIcon v-else-if="name === 'status-blocked'" :path="mdiPlay" size="1.3rem" />
```

Comment:
Why do you go in this direction now? Please use the `mdiStopCircle` and `mdiPlayCircle` icons here as we agreed to use filled circle icons for **all** status icons.

Reply:
My mistake, wasn't on purpose.